rollup.config.js

```diff
@@ -11,13 +11,17 @@ export default {
   },
   plugins: [
     babel(),
+    svelte({
```

Comment:
curios what is the benefit to move svelte plugin upward (previously I just sort alphabetically)